investment_tracking/models.py

```diff
@@ -16,9 +16,9 @@ class InvestmentTracking(models.Model):
         __str__(): Returns a string representation of the investment tracking entry.
     """
```

Comment:
Update the docstring to include the unique constraint.
Meta:
    unique constraint on (investor, startup) to prevent duplicate investments.

Reply:
Done